internal/glfwwin/wglcontext_windows.go

```
@@ -208,15 +208,15 @@ func (w *Window) choosePixelFormat(ctxconfig *ctxconfig, fbconfig_ *fbconfig) (i
 func makeContextCurrentWGL(window *Window) error {
 	if window != nil {
 		if err := wglMakeCurrent(window.context.wgl.dc, window.context.wgl.handle); err != nil {
-			_glfw.contextSlot.set(0)
+			_ = _glfw.contextSlot.set(0)
```

This should return the error

---

NVM, this cannot

---

so it should only return the error from contextSlot.set, and not from wglMakeCurrent? or should we somehow wrap both errors

---

ah, your comment hadn't loaded. my bad :P